fe/fe-core/src/main/java/org/apache/doris/load/Load.java

```diff
@@ -1064,6 +1064,12 @@ private static void initColumns(Table tbl, List<ImportColumnDesc> columnExprs,
                 // which is not nullable.
                 slotDesc.setIsNullable(true);
                 slotDesc.setColumn(new Column(realColName, PrimitiveType.VARCHAR));
+
+                boolean isBitmapType = tbl.getColumn(columnName).getType().isBitmapType();
+                if(isBitmapType){
```

Comment:
I think it is better to change the code logic like: if (isBitmapType) { xxx } else { xxx }.
Then we could sure functions like, setIsMaterialized and setIsNullable and so on, are called as expected.

Reply:
Yes, thank you. I have changed the code logic